torchvision/prototype/transforms/_misc.py

```diff
@@ -92,11 +95,15 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
 
 class Normalize(Transform):
     def __init__(self, mean: List[float], std: List[float]):
+        # datumbox: The previous documentation says taht these need to be sequencies but here we have lists. Should
+        # refactor this to ensure BC?
```

Comment:
cc @vfdev-5 / @pmeier

Reply:
I'm picking this up.

Reply:
#6490

Reply:
No this should be sequences. See #5626. We should have as loose types as possible on the transforms and dispatchers and only make them concrete before calling the kernels.

Reply:
@pmeier Just to confirm, you agree with the changes I did on #6490?

Reply:
Reviewed there.